### PR TITLE
Implement Cluster, pmctl and e2e test

### PR DESCRIPTION
**Pull request by ysksuzuki**
This PR implements

- Set up Cluster using components
- pmctl
- e2e test
- Replace monitor with QMP
- Fix some bugs

### CODE REVIEW

File: v2/version.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package v2
+
+import (
+	"runtime/debug"
+	"strings"
+)
+
+const version = "2.0.1"
```
Comment (ymmt2005) [inline v2/version.go:8]: seems strange.

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

File: v2/cmd/placemat2/sub/run.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	"github.com/cybozu-go/log"
+	"github.com/cybozu-go/placemat/v2/pkg/placemat"
+	"github.com/cybozu-go/placemat/v2/pkg/types"
+	"github.com/cybozu-go/placemat/v2/pkg/vm"
+	"github.com/cybozu-go/well"
+	"github.com/gin-gonic/gin"
+)
+
+func subMain(args []string) error {
+	rand.Seed(time.Now().UnixNano())
```
Comment (ymmt2005) [inline v2/cmd/placemat2/sub/run.go:21]: Is this garbage?

Comment (ysksuzuki) [reply v2/cmd/placemat2/sub/run.go:21]: Oops

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**